kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -1251,6 +1251,34 @@ paths:
         404:
           description: Not found
 
+  /api/streamtopology:
+    get:
+      tags:
+        - StreamTopology
+      summary: get stream topology
+      operationId: getStreamTopology
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GlobalTopology'
+
+  /api/streamtopology/dummy:
```

Comment:
Let's skip dummy endpoints in source code

Reply:
done